app/views/users/index.erb

```diff
@@ -0,0 +1,13 @@
+<h1> User index </h1>
+
+<%= button_to "Add New User", new_user_path, method: :get %>
+
+<ul> 
+    <%= @users.each do |user| %>
+    <li><%= link_to user.username, user_path(user) %><li> 
+    <br>
+    <p>end iterate</p>
```

Comment:
end iterate?

Reply:
I was trying to locate the bug issue. The server shows lines that I didn't code. reported it in the issue so we can review tomorrow.